pkg/security/probe/activity_dump.go

```diff
@@ -966,10 +966,11 @@ func (pan *ProcessActivityNode) snapshotBoundSockets(p *process.Process, ad *Act
 		return err
 	}
 
+	// sockets have the following pattern "socket:[inode]"
 	var sockets []uint64
 	for _, fd := range FDs {
-		if socketRe.MatchString(fd.Path) {
```

Comment:
I think you can remove the regexp definition now

Reply:
fixed, thanks